messages.md

```diff
@@ -384,6 +452,77 @@ to validate the signature.
 
 
 
+#### `finalize_spend`
+
+Sent by a manager to finalize their spending attempt by presenting the fully-signed spend
+transaction to the watchtowers.
+
+```json
+{
+    "method": "finalize_spend",
+    "params": {
+        "transaction": "fully signed spend transaction (PSBT format)",
+        "vault_id": "vault_uid"
+    }
+}
+```
+
+The `vault_uid` is `sha256(vault txid)`.
+
+
+#### `get_spend_validations`
+
+Sent by a manager when polling for watchtowers acknowledgement of the fully signed
+transaction spending the vault identified by `vault_id`.
+
+```json
+{
+    "method": "get_spend_validations",
+    "params": {
+        "vault_id": "vault_uid"
+    }
+}
+```
+
+The `vault_uid` is `sha256(vault txid)`.
+
+
+#### `spend_validations`
```

Comment:
Since the SS is composing multiple responses from WTs into a message here, we can't expect the WT to have signed 'this exact JSON' without first communicating with each other. I think the description in the sig fields should be more like 
""ECDSA (secp256k1) signature of this _opinion_ json with no space and 'sig:\"\"'"

Reply:
But that gets me thinking maybe the opinion needs to be mapped to the WT's pubkey (unless we want to try and fail to verify several times until we determine which opinion came from which WT).

Reply:
Yeah, this is spec'ed below but i could make it clearer. The signature is for the JSON object as sent in `spend_opinion`

Reply:
> But that gets me thinking maybe the opinion needs to be mapped to the WT's pubkey (unless we want to try and fail to verify several times until we determine which opinion came from which WT).

But it is already ?

Reply:
For reference: discussed out of band, we probably need to include the pubkey here to identify the WT from the set of messages. Will do this in a followup